scripts/pylib/twister/harness.py

```diff
@@ -41,6 +49,10 @@ def configure(self, instance):
             self.repeat = config.get('repeat', 1)
             self.ordered = config.get('ordered', True)
             self.record = config.get('record', {})
+            self.running_dir = config.get('running_dir', None)
```

Comment:
why are we defining all of this, we know where we are running and where the source directory is.

Reply:
fixed in #33227